kyaml/fn/runtime/runtimeutil/runtimeutil_test.go

```diff
@@ -1394,7 +1401,7 @@ metadata:
 		if !assert.NoError(t, err) {
 			return
 		}
-		fn := GetFunctionSpec(cfg)
+		fn, _ := GetFunctionSpec(cfg)
```

Comment:
```suggestion
		fn, err := GetFunctionSpec(cfg)
		assert.NoError(t, err)
```

Reply:
This also applies to lines 1323 and 1542. 

If you are expecting an error in one of these places,, you can do `assert.Error(t, err)` instead

Reply:
I fix it.